e2e/utils.go

```diff
@@ -1139,23 +1138,30 @@ func validateController(f *framework.Framework, pvcPath, appPath, scPath string)
 	if err != nil {
 		return err
 	}
-	// resize PVC
-	err = expandPVCSize(f.ClientSet, resizePvc, expandSize, deployTimeout)
-	if err != nil {
-		return err
-	}
-	if *pvc.Spec.VolumeMode == v1.PersistentVolumeFilesystem {
-		err = checkDirSize(app, f, &opt, expandSize)
+	if scParams["encrypted"] == strconv.FormatBool(true) {
```

Comment:
We don't need conversion here.

```suggestion
if scParams["encrypted"] == "true" {
```

Reply:
https://github.com/ceph/ceph-csi/pull/2296#discussion_r682797040
`goconst` linter complaints about it.